api/test/application/api/invite-post.js

```diff
@@ -59,6 +53,45 @@ describe(`POST /api/invite`, () => {
     expect(emailTokens.length).to.equal(1);
 
     expect(sendEmailMock.calledOnce).to.equal(true);
-  })
+  });
+
+  it(`successfully invites but does NOT create a user when provided an email address that already exists, case insensitive`, async () => {
+    const sendEmailMock = await mockSendEmail();
+    await client.load('redirect_uris');
+
+    const testEmail = 'TestEmail@example.com';
+    const testEmailDifferentCase = testEmail.toUpperCase();
+
+    await factory.create('user', {
+      email: testEmail,
+      email_lower: testEmail.toLowerCase()
+    });
+
+    let users = await knex('SIP_user').where({ email_lower: testEmail.toLowerCase() });
+    expect(users.length).to.equal(1);
```

Comment:
This assertion is a bit odd. All it's checking is whether the factory worked, which we can just assume it does.